starlight/templates/base.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+{% load i18n %}
+{% load staticfiles %}
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>
+        {% block title %}{% trans "Starlight" %}{% endblock %}
+    </title>
+
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.8/semantic.min.css" rel="stylesheet">
```

Comment:
Since we are using Semantic UI from our locally-installed environment (provided through Webpack), no need to load it via CDN here.

Reply:
I'm having trouble finding out how to load it from the locally-installed environment. I tried paths like:

<link href="/semantic/dist/semantic.min.css" rel="stylesheet" type="text/css">

But they dont seem to properly use semantic-ui